File: TransitionTest/app/src/main/java/com/example/transitiontest/SecondActivity.java
package com.example.transitiontest;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.transition.Slide;

public class SecondActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);

//        getWindow().setExitTransition(new Slide());
//        getWindow().setEnterTransition(new Slide());
    }
}

File: DataBingdingTest/app/src/main/java/com/xkf/databingdingtest/SecondActivity.kt
package com.xkf.databingdingtest

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.xkf.databingdingtest.databinding.ActivitySecondBinding

class SecondActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val binding = ActivitySecondBinding.inflate(layoutInflater)
        setContentView(binding.root)

        val imageUrl = "https://s1.ax1x.com/2020/03/31/GMgyaF.jpg"
        binding.url = imageUrl
    }
}